aplib.net-demo/Assets/Scripts/Doors/TempItemID.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+/// <summary>
+/// Temporary class that gives an ID to the player, change the ID in the editor of the player object
+/// </summary>
+public class TempItemID : MonoBehaviour
```

Comment:
```suggestion
public class TempItemId : MonoBehaviour
```
ID should be Id because we only capitalize the first letter, even if it is a abbreviation.